test/jest/victory-tooltip/victory-tooltip.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Client tests
+ */
+/* global console */
+import React from "react";
+import { VictoryTooltip } from "victory-tooltip";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("components/victory-tooltip", () => {
+  const baseProps = {
+    x: 0,
+    y: 0,
+    datum: { some: "object" },
+    index: 3,
+    active: true,
+    text: "such text, wow"
+  };
+  beforeAll(() => {
+    jest.spyOn(console, "warn").mockImplementation(() => {});
+    jest.spyOn(console, "error").mockImplementation(() => {});
+  });
+
+  it("renders nothing when not active", () => {
+    render(<VictoryTooltip {...baseProps} active={false} />);
+    const output = screen.findByText(baseProps.text);
+    expect(output).toBeDefined();
```

Comment:
This might be how the original was, but should we add a test assertion here that confirms the tooltip isn't visible? This test assertion is exactly the same as the one on line 32.

Reply:
Sweet!  I added `jest-dom`, and this became:
```
expect(output).toBeInTheDocument();
expect(output).not.toBeVisible();
```

Also, this massively helped me find a mistake in this test too!  It should be `screen.getByText`, not the async `findByText` ... phew!